docs/partners/become-a-tech-partner.md

```diff
@@ -118,3 +118,10 @@ Now that the integration is complete, you'll get the visibility you deserve! The
 
 Partners who wish to contribute paid packs must follow a few [additional onboarding steps](/docs/partners/premium-packs-process). 
 
+## 8. Suggested Timeline
+
+Here is an example of a timeline that shows a full development cycle for a pack
+
+![Edi - Frame 1 (2)](https://user-images.githubusercontent.com/85438368/163173540-bb88adbd-734c-439f-8722-a788106349b6.jpg)
```

Comment:
@edik24 Also, I cant read it, can you allow the user to click on it to zoom in?

Reply:
there is no way to click on pictures in xsoar.pan.dev.
I seperated to two pictures
let me know what you think